### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: shell_integration/windows/OCContextMenu/OCContextMenu.cpp [hunk 1]
```diff
@@ -72,7 +72,7 @@ IFACEMETHODIMP_(ULONG) OCContextMenu::Release()
 
 // Initialize the context menu handler.
 IFACEMETHODIMP OCContextMenu::Initialize(
-    LPCITEMIDLIST pidlFolder, LPDATAOBJECT pDataObj, HKEY hKeyProgID)
+    LPCITEMIDLIST, LPDATAOBJECT pDataObj, HKEY)
```
Comment (dschmidt) [inline shell_integration/windows/OCContextMenu/OCContextMenu.cpp:75]: We probably won't find a consensus here ...
but I would like to point out that I think it's a bad idea to remove parameter names here ... I would prefer `Q_UNUSED` or adding `/* */ ` around the parameter names.

File: shell_integration/windows/OCContextMenu/OCContextMenu.cpp [hunk 2]
```diff
@@ -88,7 +88,7 @@ IFACEMETHODIMP OCContextMenu::Initialize(
         HDROP hDrop = static_cast<HDROP>(GlobalLock(stm.hGlobal));
         if (hDrop) {
             UINT nFiles = DragQueryFile(hDrop, 0xFFFFFFFF, NULL, 0);
-            for (int i = 0; i < nFiles; ++i) {
+            for (UINT i = 0; i < nFiles; ++i) {
```
Comment (dschmidt) [inline shell_integration/windows/OCContextMenu/OCContextMenu.cpp:91]: Why in caps?

**Review by dschmidt — APPROVED** [review]

File: shell_integration/windows/OCContextMenu/OCContextMenu.cpp (hunk 2 above)
Comment (TheOneRing) [reply shell_integration/windows/OCContextMenu/OCContextMenu.cpp:91]: I just use the same type as nFIles

File: shell_integration/windows/OCContextMenu/OCContextMenu.cpp (hunk 1 above)
Comment (TheOneRing) [reply shell_integration/windows/OCContextMenu/OCContextMenu.cpp:75]: Anyone who touches the code will need to open the doc anyhow

**Pull request merged**